app/src/main/java/com/nextcloud/talk/ui/dialog/SetStatusDialogFragment.kt

```diff
@@ -177,6 +176,8 @@ class SetStatusDialogFragment :
 
         setupGeneralStatusOptions()
 
+        binding.emoji.setText(getString(R.string.default_emoji))
```

Comment:
please only set the default emoji if

`currentStatus?.icon == null`

otherwise it won't show the current emoji